src/main/java/seedu/address/model/InfoHandler.java

```diff
@@ -230,24 +253,47 @@ public VisitBook generateVisitsByPerson(Id personId) {
     }
 
     /**
-     * Generates a list of visits that are associated with the given list of location Ids.
-     * @param locationIds List of location Ids.
-     * @return List of visits that are associated with the location Ids.
+     * Generates a list of visits that are associated with the given visits. This is an intermediate method for the
+     * generatePeopleCommand. After the visits of an infected person have been found, this method finds all other
+     * visits by other people that are associated with the infected person's visits. A visit is considered associated
+     * if the location and date of visit are the same.
+     * @param visitBook Visits provided.
+     * @return List of visits that are associated with the given visits.
      */
-    public VisitBook generateVisitsByLocationIds(List<Id> locationIds) {
+    public VisitBook generateAssociatedVisits(VisitBook visitBook) {
```

Comment:
Intuitively, its hard to think of visits being associated with one another. I would suggest using a better word or add more details to the method name.

Reply:
Renamed!